d2cli/main.go

```diff
@@ -88,6 +89,11 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	timeoutFlag, err := ms.Opts.Int64("D2_TIMEOUT", "timeout", "", 0, "the number of seconds before d2 will timeout. default is 60s.")
```

Comment:
Here it says `60s`, but a few lines below it is `ctx, cancel := log.WithTimeout(ctx, time.Minute*2)`.
I think 60s is the default for TALA.
Also, for default I'd say it should be `60` instead of `60s` otherwise users may think the suffix is required

Reply:
hmm updated the description, and updated the Layout timeout (for any plugin) to match.
If any Layout will timeout after 60s, then any d2 run that has a Layout longer than 60s will fail. It is possible for layouts smaller than 60s to add up to > 2 minutes and then hit the d2 timeout, but there's no reason to prematurely timeout a layout that takes 61s if we are okay with d2 running for 2 minutes. It seems fine to spend 99.9% of the time on Layout (vs any d2 overhead) as long as it completes within the 2 minutes.